test/fixtures/internal.js

```diff
@@ -842,11 +842,8 @@ Fixtures.FormRendererOptions = {
       project_id: 'dummy_val',
       response_fields: [
         {
-          id: 34,
-          label: "",
-          required: false,
-          blind: false,
-          admin_only: false,
+          id: 'id',
+          required: true,
```

Comment:
I've updated our fixtures to match the data that Screendoor passes for identification fields